code/src/styles.js

```diff
@@ -0,0 +1,194 @@
+import styled from 'styled-components'
+import { createGlobalStyle } from 'styled-components'
+
+export const devices = {
+    mobile: "(min-width: 375px)",
+    tablet: "(min-width: 768px)",
+    desktop: "(min-width: 1025px)",
+}
+
+export const GlobalStyle = createGlobalStyle`  
+  body {
+    margin: 0;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-image: url(../images/backgroundPink.jpg);
+    // background-color: lightpink;
+    background-size: auto;
+    background-repeat: no-repeat;  
+    background-position-x: 9%; 
+    display: flex;
+    justify-content: center;
+    font-family: 'Ubuntu', sans-serif;
+  }
+
+  h1 {
+      font-size: 25px;
+      text-align: center;
+      margin: 10px 20px 0px 20px;
+      border-bottom: solid black 1px;
+    }
+
+  h3 {
+      font-size: 15px;
+      text-align: center;
+      color: black;
+      font-style: italic;
+    }
+
+  p {
+      text-align: center;
+      border: solid 1px #EA99AA;
+      border-radius: 5px;
+    }
+
+  section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+//   [type="checkbox"]{
+//       opacity: 0;
+//     }
+
+//   [type="checkbox"] + label {
+//     poistion: relative;
+//     padding-left: 30px;
+//     cursor: pointer;
+//     display: inline-block;
+//     color: #666;
+//     line-height: 25px;
+//   }
+
+//   [type="checkbox"] + label::before {
+//     content: "";
+//     position: absolute;
+//     left: 56px;
+//     top: 51px;
+//     width: 18px;
+//     height: 18px;
+//     outline: 2px solid #aaa;
+//     background: #fff;
+//   }
+
+//   [type="checkbox"]:checked + label::before {
+//     content: "";
+//     position: absolute;
+//     left: 56px;
+//     top: 51px;
+//     width: 18px;
+//     height: 18px;
+//     outline: 2px solid #5fd25f;
+//     background: #fff;
+//   }
+
+//   [type="checkbox"]:checked + label::after {
+//     content: "";
+//     position: absolute;
+//     left: 56px;
+//     top: 51px;
+//     width: 18px;
+//     height: 18px;
+//     outline: 2px solid #5fd25f;
+//     background: #fff;
+//   }
+`
```

Comment:
Perhaps you saved this for the future, but a suggestion would be to delete the out commented code to make it look a little more coherent, or maybe put it on the bottom of the pabe? :)